modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/rel/IgniteSortedIndexSpool.java

```diff
@@ -168,6 +168,6 @@ public RelOptCost computeSelfCost(RelOptPlanner planner, RelMetadataQuery mq) {
 
         IgniteCostFactory costFactory = (IgniteCostFactory) planner.getCostFactory();
 
-        return costFactory.makeCost(rowCnt, cpuCost, 0, totalBytes, 0);
+        return costFactory.makeCost(rowCnt, cpuCost, 0, totalBytes, 0).multiplyBy(0.01);
```

Comment:
whats the purpose for such a change ? i remove it and all sql-engine tests are ok. possibly some unstable tests or after main rebase it`s all ok there ?

Reply:
Thanks, removed